controllers/defaults.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"os"
+)
+
+var (
+	DefaultValMap = map[string]string{
+		"OPERATOR_NAMESPACE":    "openshift-storage",
+		"TOPOLVM_CSI_IMAGE":     "quay.io/topolvm/topolvm:0.10.3",
+		"CSI_REGISTRAR_IMAGE":   "k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.3.0",
+		"CSI_PROVISIONER_IMAGE": "k8s.gcr.io/sig-storage/csi-provisioner:v3.0.0",
+		"CSI_ATTACHER_IMAGE":    "k8s.gcr.io/sig-storage/csi-attacher:v3.3.0",
+		"CSI_SNAPSHOTTER_IMAGE": "k8s.gcr.io/sig-storage/csi-snapshotter:v4.2.0",
```

Comment:
- we'll never need attacher and snapshotter isn't being used as of now

Reply:
The topolvm makefile is using the csi attacher which is why I added it:
```EXTERNAL_PROVISIONER_SRC  = $(SRC_ROOT)/external-provisioner
NODE_DRIVER_REGISTRAR_SRC = $(SRC_ROOT)/node-driver-registrar
EXTERNAL_ATTACHER_SRC     = $(SRC_ROOT)/external-attacher
EXTERNAL_RESIZER_SRC      = $(SRC_ROOT)/external-resizer
LIVENESSPROBE_SRC         = $(SRC_ROOT)/livenessprobe```

Reply:
- there's no separate liveness probe iirc, same hypertopolvm handles that

Reply:
@nbalacha 

1. pls add the liveness probe
2. attacher can be [removed](https://github.com/topolvm/topolvm/blob/6697490595efd7cfa78bffc6b7a0581b33c18bd2/CHANGELOG.md#changed-3)

Reply:
Done.